<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Extras</class>
 <widget class="QDialog" name="Extras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>552</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>529</width>
    <height>552</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Extras</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>419</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Live Earth Wallpaper</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Live earth wallpaper automatically configures your desktop wallpaper
to be a live earth background that updates every Â½ hour.
Internet connection required.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icons.qrc">:/icons/Pictures/world_sunlight_map_rectangular.jpg</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>331</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Activate Live Earth Wallpaper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Deactivate Live Earth Wallpaper</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>419</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Lock Screen Background</string>
    </attribute>
    <widget class="QRadioButton" name="use_current_radioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>511</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Use current desktop wallpaper</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>CHANGE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="image_from_disk_lock">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>511</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Select Image From Disk</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="preview_lock">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>129</y>
       <width>230</width>
       <height>143</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>360</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>REVERT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>245</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icons.qrc">:/icons/Pictures/screen.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>use_current_radioButton</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>image_from_disk_lock</zorder>
    <zorder>pushButton_10</zorder>
    <zorder>pushButton_12</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>preview_lock</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>109</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
